lib/client.js

```diff
@@ -40,6 +40,8 @@ function httpRequest(client, endpointPath, opts, cb) {
       return cb(err)
     if (body.error)
       return cb(new Error(body.error))
+if (body.status === 'error')
```

Comment:
The change looks good. Can you please fix the alignment of this `if` statement? I'll merge it in after that. Thanks.